.github/workflows/mta-e2e.yaml

```diff
@@ -98,6 +98,13 @@ jobs:
         run: |
           # deploy the manifests created by the  ${{ steps.build-image.outputs.image }}"
           kubectl apply -f manifests
+
+          ###### workaround till https://issues.redhat.com/browse/FLPATH-892 is solved
+          sleep 2
```

Comment:
why not updating the manifest file (with `sed`/`yq`) before applying it to the cluster?
this way you would not need to delete the deployment after the patch (because of [FLPATH-893](https://issues.redhat.com/browse/FLPATH-893) that you just raised)

Reply:
I wanted to avoid `sed` because I need to add and element. I'll give `yq` a try

Reply:
done

Reply:
I'm afraid yq isn't present on the runner machine? https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md

Reply:
oh it is there according to the page